grpc/src/compiler/go_generator.cc

```diff
@@ -0,0 +1,435 @@
+/*
```

Comment:
To allow future updates from grpc-go, it is important that this file remains unchanged. If you can factor out those changes such that they happen in different files, or if there are ProtoBuf specific things in this file, they should be factored out with a PR to grpc-go